### PR TITLE
some ingredients that are sources of fodmap

### PR DESCRIPTION
**Pull request by stephanegigandet**
Some missing ingredients that are fodmap sources reported by @Etatdesprit

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -39796,6 +39801,15 @@ es:Lichis deshuesados
...
 
+<en:fruit
+en:Rambutan
+fr:Ramboutan, Litchi chevelu
+
+<en:fruit
+en:longan
+fr:longane, longani, oeil du dragon
+
+<en:Litchi
```
Comment (Ban3) [inline taxonomies/ingredients.txt:39812]: Ingredient seems to be missing here.

Comment (Etatdesprit) [reply taxonomies/ingredients.txt]: First Change Thank's

**Comment by stephanegigandet** [comment]
@Etatdesprit : there's an issue with your change, it looks like you removed all the content of the ingredients.txt file and replaced it with:

````
[
  {
      "percent_min" : 100,
      "vegetarian" : "yes",
      "vegan" : "yes",
      "id" : "en:wheat",
      "percent_max" : 100,
      "text" : "Froment",
      "rank" : 1
   }
]

**Comment by stephanegigandet** [comment]
@Etatdesprit : I'm reverting your commits, I don't know what happened, maybe some copy/paste issue.

**Pull request merged**